### PR TITLE
Feature/statsd level

### PR DESCRIPTION
**Pull request by julsemaan**
# Description

Allow to set a level of importance to statsd timer to reduce the amount of whisper DBs while keeping fine grained timers, all of that without restarting a single service (looks too good to be true...)
# Impacts

Statsd timers and anything that uses them
# Delete branch after merge

YES

### CODE REVIEW

**Comment by julsemaan** [comment]
@jrouzierinverse can you make sure I didn't go crazy with that one

File: lib/pf/db.pm [hunk 1]
```diff
@@ -177,7 +177,7 @@ sub get_db_handle {
 
 sub db_data {
     my ($from_module, $module_statements_ref, $query, @params) = @_;
-    my $timer = pf::StatsD::Timer->new({ 'stat' => called() . ".$query", sample_rate => 0.1});
+    my $timer = pf::StatsD::Timer->new({ 'stat' => called() . ".$query", sample_rate => 0.1, level => 9});
```
Comment (louismunro) [inline lib/pf/db.pm]: Maybe a bit high here? 
We may want some of that good stuff without going all the way to 11.

File: lib/pf/db.pm [hunk 2]
```diff
@@ -214,7 +214,7 @@ sub db_data {
 # afterwards: remove export of $<module>_db_prepared and <module>_db_prepare()
 sub db_query_execute {
     my ($from_module, $module_statements_ref, $query, @params) = @_;
-    my $timer = pf::StatsD::Timer->new({ 'stat' => called() . ".$query", sample_rate => 0.1});
+    my $timer = pf::StatsD::Timer->new({ 'stat' => called() . ".$query", sample_rate => 0.1, level => 9});
```
Comment (louismunro) [inline lib/pf/db.pm]: Same here. Can we say 7 or 8?
And how about that spoon?

**Comment by jrouzierinverse** [comment]
You a few redundant  "sample_rate => 0.1" in the code.

Nothing a
 `perl -pi -e's/sample_rate => 0.1,//' $(grep -PRl 'sample_rate => 0.1,' lib/ html/)` 
could not fix.

Outside of that the code looks solid

File: lib/pf/db.pm (hunk 1 above)
Comment (julsemaan) [inline lib/pf/db.pm]: as discussed verbally, it isn't :)

File: lib/pf/db.pm (hunk 2 above)
Comment (julsemaan) [inline lib/pf/db.pm]: as discussed verbally, it isn't :)

**Comment by julsemaan** [comment]
That code isn't going to reboot my machine isn't ?

**Comment by jrouzierinverse** [comment]
Run it and find out

**Pull request merged**